site/app.js

```diff
@@ -38,6 +39,7 @@ app.use(logger('combined', {stream: accessLogStream}));
 // ----------------------
 app.use('/', routes);
 app.use('/users', users);
+app.use('/announcement', announcement);
```

Comment:
Check [here](https://github.com/amc-ucdenver/website/wiki/REST-Calls) for what your endpoint should look like. Right now, you have `/announcement` but according to the spec, it should be `/articles`. We don't want to have mismatching calls and endpoints.